connect-button/src/main/java/com/ifttt/connect/Connection.java

```diff
@@ -52,22 +52,28 @@
     public final String url;
     public final List<Service> services;
 
-    @Nullable @Json(name = "cover_image") public final CoverImage coverImage;
-    @Json(name = "value_propositions") public final List<ValueProposition> valuePropositions;
+    @Nullable public final CoverImage coverImage;
+
+    /**
+     * @deprecated Use {@link #features} instead.
+     */
+    @Deprecated public final List<ValueProposition> valuePropositions = Collections.emptyList();
```

Comment:
Maybe this is a bit aggressive? We could consider filling `valuePropositions` with Feature data as they are somewhat compatible, so that the transition wouldn't be sudden. Not sure if it's worth it.

Reply:
yes I think features has two additional fields over value props, I'm open to both approaches. 

But this may be cleaner long-term, since we will be using the new `Feature` data model.